frontend/src/container/LogDetailedView/FieldRenderer.styles.scss

```diff
@@ -18,6 +18,7 @@
 
 	.tags {
 		display: flex;
-		gap: 8;
+		flex-wrap: wrap;
```

Comment:
we should ideally avoid `flex-wrap` as there can be \n and other formatters in the log lines but anyways we currently do not take this into account for raw and other views as well

Reply:
This is only for the tags in attributes key columns @vikrantgupta25